CONTRIBUTING.md

```diff
@@ -238,6 +238,19 @@ need to "save" the changes.
    - This will modify the appropriate patches based on your commits.
 1. PR your modified patch file(s) back to this repository.
 
+## Rebasing PRs
+Steps to rebase a pr to include the latest changes from `master`.
+
+1. Pull latest changes from upstream's master; `git checkout master && git pull`.
```

Comment:
We use Markdown's magic list numbering stuff all over this file already, so use only `1.` for all entries here.